### ISSUE TITLE
The result of incremental class is 0.

### ISSUE THREAD

**Issue by Zzsf11** [schuy1er/EWF_official#5]
Hi, I find this work to be both elegant and easy to understand, so I've attempted to apply it to my dataset. However, I've noticed that the incremental class values are close to zero. I also tried to reproduce the VOC results and encountered a similar issue. 

I'm quite confused by this and would appreciate any explanations or solutions you might offer.

My dataset result:
![Untitled](https://github.com/schuy1er/EWF_official/assets/52969669/b1157610-8575-4bd5-8883-497403e6b6d0)

VOC result:
![_cgi-bin_mmwebwx-bin_webwxgetmsgimg__ MsgID=4984237477748255634 skey=@crypt_14c47ce3_0ad5c62d377a7cffb408e60604fa7eb7 mmweb_appid=wx_webfilehelper](https://github.com/schuy1er/EWF_official/assets/52969669/f9450925-b66d-4c4f-a91b-45689abf17ea)

**Comment by schuy1er** [comment]
Hi, have you tried to reproduce PLOP? How is its performance?

**Comment by Zzsf11** [comment]
![Untitled (1)](https://github.com/schuy1er/EWF_official/assets/52969669/80e0e015-0c56-45ca-b146-32459525099f)
yes, But I only try it on my own dataset. It seems to have similar problem.

**Comment by schuy1er** [comment]
If PLOP does not perform well on this dataset, then EWF may also have difficulty in achieving good results on new classes, because EWF is used in conjunction with distillation (EWF emphasizes more on alleviating forgetting, and the improvement for new classes actually appears more in multi-step fine-tuning: less forgetting in the last few stages after training the new class). My suggestion is that you can try using MiB, which is a more relaxed constraint than PLOP. In addition, you can try to adjust the weighted coefficient alpha of EWF for averaging, but this is based on the premise that you can first learn new classes.

**Comment by schuy1er** [comment]
Plus, there is a bug that may cause performance differences. You can try to modify it, I hope it can help you. 
[https://github.com/zhangchbin/RCIL/commit/83ec4191b2056b314b5409b701e2e8a0ae0709f6](url)

**Comment by Zzsf11** [comment]
Thank for your explain! 
I have attempted on MiB for my dataset:
![Untitled (3)](https://github.com/schuy1er/EWF_official/assets/52969669/37fe2791-9c6a-461e-9946-01be845373d5)
The result is  ......( so bad ).
As I visualize the incremental class, I've noticed that the primary issue involves the correct masks being assigned to the wrong classes. It seems that some incremental classes are very similar to the base classes, which could be causing the poor results. So I think maybe some methods that enhance classify ability could improve that.
